librz/core/cprint.c

```diff
@@ -158,21 +158,289 @@ RZ_API RZ_OWN char *rz_core_esil_of_hex(RzCore *core, ut8 *hex, int len) {
 	return NULL;
 }
 
+/**
+ * \brief Prints \p str to the RzCons, and ownership is transferred into.
+ */
+static inline bool rz_cons_print_own(char *str) {
+	if (!str) {
+		return false;
+	}
+	rz_cons_print(str);
+	free(str);
+	return true;
+}
+
```

Comment:
I think this is not really necessary.
Also, probably if the strings you pass to this function are NULL you may want to print a specific error message to the users, telling them X is wrong.